### PR TITLE
Edited for loop to print out only even numbers

### LINKED ISSUE #6: Edit for loop
Edit for loop to print out only even numbers.

### PR DESCRIPTION
**Pull request by mariomaxmachine**
resolves #6

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -8,4 +8,5 @@ def sayHello(name):
     x = 10
 
     for i in range(1, x + 1):
-        print(i)
\ No newline at end of file
+        if i % 2 == 0:
+            print(i)
```
Comment (asarwinski) [inline test.py:12]: IDEALNE ROZWIĄZANIE

**Review by asarwinski — COMMENTED** [review]
DOBRA ROBBOTA

File: test.py (hunk 1 above)
Comment (asarwinski) [inline test.py:12]: Add some comments

Comment (asarwinski) [inline test.py:12]: Test

**Review by asarwinski — CHANGES_REQUESTED** [review]

**Comment by mariomaxmachine** [comment]
yes

**Review by asarwinski — APPROVED** [review]
DOBRA ROBOTA

**Pull request merged**